### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/cookie-param-models.md`

### PR DESCRIPTION
**Pull request by ceb10n**

### CODE REVIEW

**Review by nazarepiedady — APPROVED** [review]

File: docs/pt/docs/tutorial/cookie-param-models.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+{!> ../../../docs_src/cookie_param_models/tutorial002.py!}
+```
+
+////
+
+Se o cliente tentar enviar alguns **cookies extras**, eles receberão um retorno de **erro**.
+
+Coitados dos banners de cookies com todo o seu esforço para obter o seu consentimento para a <abbr title="Isso é uma outra piada. Não preste atenção em mim. Beba um café com o seu cookie. ☕">API rejeitá-lo</abbr>. 🍪
+
+Por exxemplo, se o cliente tentar enviar um cookie `santa_tracker` com o valor de `good-list-please`, o cliente receberá uma resposta de **erro** informando que o <abbr title="O papai noel desaprova a falta de biscoitos. 🎅 Ok, chega de piadas com os cookies.">cookie `santa_tracker` is not allowed</abbr>:
```
Comment (alejsdev) [inline docs/pt/docs/tutorial/cookie-param-models.md:139]: I think this is a typo (exxemplo) :eyes:

Comment (ceb10n) [reply docs/pt/docs/tutorial/cookie-param-models.md:139]: You're right!! Thank you @alejsdev 😄

Comment (ceb10n) [reply docs/pt/docs/tutorial/cookie-param-models.md]: Done 😃

**Comment by alejsdev** [comment]
Thank you! @ceb10n :nerd_face:

**Pull request merged**